docs/reference-manual/src/main/asciidoc/set.adoc

```diff
@@ -70,8 +69,10 @@ asadmin-options::
 
 === Operands
 
-attribute-name=value::
-  Identifies the full dotted name of the attribute name and its value.
+attribute-name=value [attribute-name=value]::
+  Identifies the full dotted name of the attribute name and its value. 
+  Multiple attribute=value pairs can be provided, separated by a space (,).
```

Comment:
> by a space (,).

Does this indicate that a comma can replace a space?

Reply:
This is my mistake, I'll fix it. I first thought that values should be separated by commas, but then realized it's by space. And I forgot to remove the comma in parenthesis.